Documentation/ApiOverview/Database/DatabaseStructure/Index.rst

```diff
@@ -1,124 +1,135 @@
-.. include:: /Includes.rst.txt
-.. index::
-   Database; Structure
-   Tables
-.. _database-structure:
-.. _database-structure-requirements:
+..  include:: /Includes.rst.txt
+..  index::
+    Database; Structure
+    Tables
+..  _database-structure:
+..  _database-structure-requirements:
 
 ==================
-Database Structure
+Database structure
 ==================
 
-The database tables used by TYPO3 can be divided into two
-rough categories:
+The database tables used by TYPO3 can be roughly divided into two categories:
 
-- Tables that are used by the system internally and are invisible to backend
-  users (eg. `be_sessions`, `sys_registry`, cache related tables). There are
-  often dedicated PHP API's in the Core extension to manage entries of these
-  tables, for instance the :ref:`Caching framework API <caching>`.
+-   Tables that are used internally by the system and are invisible to backend
+    users (for example, :sql:`be_sessions`, :sql:`sys_registry`, cache-related
+    tables). In the Core extension, there are often dedicated PHP APIs for
+    managing entries in these tables, for instance, the :ref:`caching framework
+    API <caching>`.
 
-- Tables that can be managed via the TYPO3 backend, are shown in the List
-  module and can be edited using :ref:`FormEngine <FormEngine>`.
+-   Tables that can be managed via the TYPO3 backend are shown in the
+    :guilabel:`List` module and can be edited using the :ref:`FormEngine
+    <FormEngine>`.
 
 There are certain requirements for such managed tables:
 
-- The table must be configured in the :doc:`global TCA array <t3tca:Index>`.
-  This will tell TYPO3 things like the table name,
-  features you have configured, the fields of the table and how to
-  render these in the backend, relations to other tables, etc.
+-   The table must be configured in the :doc:`global TCA array <t3tca:Index>`.
+    This tells TYPO3 things like the table name, features you have configured,
+    the fields in the table and how they should be rendered in the backend,
+    relations to other tables, and so on.
 
-- The table must contain at least these fields:
+-   The table must contain at least these fields:
 
-  - "uid" - an auto-incremented integer, PRIMARY key, for the table,
-    containing the *unique ID* of the record in the table.
+    -   :sql:`uid` - an auto-incremented integer and primary key for the table,
+        containing the *unique ID* of the record in the table.
 
-  - "pid" - an integer pointing to the "uid" of the page (record from
-    "pages" table) to which the record belongs.
+    -   :sql:`pid` - an integer pointing to the :sql:`uid` of the page (record
+        from :sql:`pages` table) to which the record belongs.
 
-  - other typical fields include:
+-   Other typical fields include:
```

Comment:
I would prefere some headlines for structure over nested lists. We can link to headlines and they are easier to detect visually and easier to navigate with screenreaders